### PR TITLE
Reduce accepted cyclomatic complexity from 32 to 25

### PR DESCRIPTION
**Pull request by blowekamp**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -33,4 +33,4 @@ local_scheme = "dirty-tag"
 [tool.flake8]
 max-line-length = 88
 ignore = ['E203', 'W503']
-max_complexity = 32
+# max_complexity = 32
```
Comment (jcfr) [inline pyproject.toml:36]:
```suggestion
```

**Comment by jcfr** [comment]
Suggested commit title:

```
ENH: Reduce accepted cyclomatic complexity from 32 to 10
```

File: pyproject.toml (hunk 1 above)
Comment (jcfr) [reply pyproject.toml:36]: I suggest to be explicit.

**Comment by jcfr** [comment]
I will revisit the integration of `flake8-pyproject` and pre-commit, something seems incorrect.

**Review by dave3d — APPROVED** [review]

**Comment by blowekamp** [comment]
> I will revisit the integration of `flake8-pyproject` and pre-commit, something seems incorrect.

I'm not sure what appears incorrect.

I just set the value to 24, which triggered:
```
codespell.py:384:1: C901 'main' is too complex (25)
```
as expected. I will update to 25, and it is expected to pass.

**Pull request merged**